### PR TITLE
feat(committee): introduce VotingPeriodRange

### LINKED ISSUE #428: Replace `Days` type in committee with range
## Issue summary

- currently, we check the validity of a votingperdio like this `in [T::Days, T::Days * 4]`, we should replace this with a `T::ValidVotingPeriod` type that includes both thresholds and acts as a range, similar to the `StakingCap` type of the remote asset manager

## Other information and links

-

### PR DESCRIPTION
**Pull request by clearloop**
## Changes

- use `VotingPeriodRange` and remove `Days` in the config of pallet committee
- updates initial VotingPeriod
-

## Tests

```
CI
```

## Issues

- Closes #428

### CODE REVIEW

File: pallets/committee/src/lib.rs [hunk 1]
```diff
@@ -516,11 +517,8 @@ pub mod pallet {
 		pub fn set_voting_period(origin: OriginFor<T>, voting_period: T::BlockNumber) -> DispatchResult {
 			T::ApprovedByCommitteeOrigin::ensure_origin(origin)?;
 
-			let days = T::Days::get();
-			let weeks = days.saturating_mul(7u32.into());
-
 			ensure!(
-				weeks < voting_period && voting_period < weeks.saturating_mul(4u32.into()),
+				T::VotingPeriodRange::min() < voting_period && voting_period < T::VotingPeriodRange::max(),
```
Comment (mattsse) [inline pallets/committee/src/lib.rs:521]: this should be `[min, max]` not `]min, max[` so that min and max are still in bounds

**Review by mattsse — CHANGES_REQUESTED** [review]
needs additional tests that update the period with the boundary values

File: pallets/committee/src/tests.rs [hunk 2]
```diff
@@ -526,20 +526,19 @@ fn propose_constituent_works() {
 fn can_set_voting_period() {
```
Comment (clearloop) [inline pallets/committee/src/tests.rs:526]: > needs additional tests that update the period with the boundary values

**Review by mattsse — APPROVED** [review]
lgtm
